relay-general/src/pii/builtin.rs

```diff
@@ -135,6 +142,13 @@ declare_builtin_rules! {
             text: "[mac]".into(),
         }),
     };
+    "@mac:hash" => RuleSpec {
+        ty: RuleType::Mac,
+        redaction: Redaction::Hash(HashRedaction {
+            algorithm: HashAlgorithm::HmacSha1,
```

Comment:
Isn't this equivalent to `HashRedaction::default()`?

Reply:
you're right